SRC/KEYB.H

```diff
@@ -32,6 +32,7 @@ namespace keyb
     const SDL_Scancode CHR_0 = SDL_SCANCODE_0;
     const SDL_Scancode CHR_F = SDL_SCANCODE_F;
     const SDL_Scancode CHR_Y = SDL_SCANCODE_Y;
+    const SDL_Scancode F5 = SDL_SCANCODE_F5;
```

Comment:
You could remove CHR_F as it's not used anymore

Reply:
Check, done.